spec/turn_spec.rb

```diff
@@ -0,0 +1,43 @@
+require './lib/card'
+require './lib/turn'
+
+
+
+RSpec.describe Turn do
+  it "exists" do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+    expect(turn).to be_instance_of(Turn)
+  end
+
+
+  it 'has a card' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau",card)
+    expect(turn.card).to eq(card)
+  end
+
+  it "has a guess" do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+    expect(turn.guess).to eq("Juneau")
+  end
+  #
+   it "has correct guess" do
+      card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+      turn = Turn.new("Juneau", card)
+      expect(turn.correct?).to eq(true)
+    end
+  #
+    it "has feedback" do
+       card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+       turn = Turn.new("Juneau", card)
+       expect(turn.feedback).to eq("Correct!")
+     end
+
+     it "has card w. answer equal to mercury" do
+        card = Card.new("Which planet is closest to the sun?", "Mercury", :STEM)
+        turn = Turn.new("Saturn",card)
+        expect(turn.card).to eq(card)
+      end
```

Comment:
also test for if a incorrect guess is taken -- make sure `correct?` returns false, and that the feedback is "Incorrect".